app/controllers/replies.rb

```diff
@@ -0,0 +1,11 @@
+class Chitter < Sinatra::Base
+  get '/reply' do
+    erb :'replies/reply'
+  end
+
+  post '/reply' do
+    reply = Reply.create(text: params[:reply], replier: current_user.username, time: Time.now.asctime)
```

Comment:
Metrics/LineLength: Line is too long. [102/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)